ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewBackgroundDrawable.java

```diff
@@ -123,6 +126,21 @@ public int getOpacity() {
     return ColorUtil.getOpacityFromColor(ColorUtil.multiplyColorAlpha(mColor, mAlpha));
   }
 
+  /* Android's elevation implementation requires this to be implemented to know where to draw the shadow. */
+  @Override
+  public void getOutline(@NonNull Outline outline) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+      float borderRadius = (!CSSConstants.isUndefined(mBorderRadius) && mBorderRadius > 0)
+              ? mBorderRadius
+              : 0;
+      float extraRadiusFromBorderWidth = (mBorderWidth != null)
```

Comment:
Is this needed? How does the shadow look without it? Maybe we just need higher elevation?